coderd/prometheusmetrics/insights/metricscollector.go

```diff
@@ -143,18 +163,62 @@ func (mc *MetricsCollector) Collect(metricsCh chan<- prometheus.Metric) {
 		return // insights data not loaded yet
 	}
 
+	// Custom apps
+	for _, appRow := range data.apps {
+		displayName := appRow.DisplayName.String
+		if displayName == "" { // just in case Prometheus complains about missing label value
+			displayName = "unknown"
```

Comment:
I think empty is fine 🤔, how about also including slug in the labels so that they can be differentiated in the absence of a display name?

Reply:
Done